### ISSUE TITLE
[A11y]The Keyboard focus is not visible on selected item from the table.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7881]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1063683
Repro Steps:
1. Hit the URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Search for a key term (say “json”) from search box.
5. Click on "Newtonsoft.json" control and hit enter.
6. Navigate to "Version History" control and hit enter.
7. Turn on high contrast.
8.Check if in high contrast the row that is selected(highlighted),the same highlighted row is also visible in high contrast while navigating using Tab key.

Actual:
The keyboard focus on selected item from the table is not clearly visible in high contrast black/white mode.

Similar issue have been observed in below scenarios : 
1).Json ->OwnerSection
Repro Steps:
1. Hit the URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Search for a key term (say “json”) from search box.
5. Click on "Newtonsoft.json" control and hit enter.
6. Navigate to "Owners" section present on bottom right of the screen.
7. Check if the image of "newtonsoft" is visible in high contrast black mode.
1063683_A11y_[BM]NuGetService_OwnerSection_HighContrast.PNGBug_1063683_A11y_[BM]NuGetService_OwnerSection_HighContrast.PNG

Expected:
The focus on the selected item from the table should be clearly visible in high contrast mode.

Notes:
I was actually unable to reproduce case 1 at all. The table highlights keyboard focus correctly without issue. The only possible issue is that the light blue that indicates which version page we are on is not visible in high contrast.
For case 2, I'm not sure what action we are expected to take for static images; I'm not sure we can be expected to apply high contrast settings to static images.

**Comment by ryuyu** [comment]
So it appears the issue is that the selected row is not visible in high contrast (having nothing to do with tab navigation).
